docs/guide/data_migration/add_data_migration_matcher.md

```diff
@@ -0,0 +1,53 @@
+# Add data migration matcher
+
+[Matchers in data migrations](exporting_and_importing_data.md#match-property) enable you to select which data from the Repository to export.
+
+In addition to the built-in matchers, you can create custom matchers for content.
+
+The following example create a matcher for Section identifiers.
```

Comment:
Matchers will be `FilteringCriterion`s soon (this is because migration process should not actually rely on SearchEngine). So I guess we should stress out that the Normalizer that will be created needs to `denormalize` into an instance of one.

See https://github.com/ibexa/migrations/pull/279